target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-unifi-ap-hd.dts

```diff
@@ -0,0 +1,312 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq8064-v2.0.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Ubiquiti UniFi AP HD";
+	compatible = "ubnt,unifi-ap-hd", "qcom,ipq8064";
+
+	aliases {
+		mdio-gpio0 = &mdio0;
+
+		led-boot = &led_dome_white;
```

Comment:
no extra spaces needed, just put in alphabetical order

Reply:
Changed.